### ISSUE TITLE
Support PATCH verb

### ISSUE THREAD

**Issue by mciszczon** [FrancescoSaverioZuppichini/API-Class#4]
Would be nice if this nice little package allowed to use PATCH method on resources. Should be easy to do, I might make a PR later today.

**Comment by FrancescoSaverioZuppichini** [comment]
It would be really nice if you can do a PR 👍

**Comment by mciszczon** [comment]
Thanks for a lightning response!

I added a new method:

```js
endpoints.patch  = ({ id }, toPatch) => axios.patch(`${resourceURL}/${id}`, toPatch)
```

Then in the React Component I execute it like this:

```js
myApi.endpoints.products.patch({ id: this.props.match.params.id }, { property: new_value })
```

I get an error however:

**ReferenceError: resourceURL is not defined**
`createBasicCRUDEndpoints/endpoints.patch`
`node_modules/api-class/distribution/API.js:74`
```js
  71 | 
  72 | endpoints.patch = function (_ref5, toPatch) {
  73 |   var id = _ref5.id;
> 74 |   return axios.patch(resourceURL + '/' + id, toPatch);
  75 | };
  76 | 
  77 | endpoints.delete = function (_ref6) {
```

I don't really get why this is not defined, when it should be by `const resouceURL`. All other methods work fine. I am not good in JS & React, really. Will be greateful for help.

**Comment by FrancescoSaverioZuppichini** [comment]
I have just merged a PR from a user that changes the name of `resouce` to `resource` since I made a type. I can suggest the follow

1) Pull again from master
2) Merge your code 
3) Try again 

The problem is 99% that you forked before I accept that PR (https://github.com/FrancescoSaverioZuppichini/API-Class/pull/1)

**Comment by mciszczon** [comment]
Fixed with #5.
